src/main/java/io/cdap/plugin/servicenow/apiclient/ServiceNowTableAPIClientImpl.java

```diff
@@ -326,9 +187,12 @@ public List<Map<String, Object>> parseResponseToResultListOfMap(String responseB
   private String getErrorMessage(String responseBody) {
     try {
       JsonObject jo = gson.fromJson(responseBody, JsonObject.class);
-      return jo.getAsJsonObject(ServiceNowConstants.ERROR).get(ServiceNowConstants.MESSAGE).getAsString();
+      return String.format("%s:%s",
+                           jo.getAsJsonObject(ServiceNowConstants.ERROR).get(ServiceNowConstants.MESSAGE).getAsString(),
```

Comment:
didn't get to see `null` for any of these fields during testing. Added null checks for all fields just in case. e63c214cf864e664ca24072f21c719c1a777b9c4